examples/tv-casting-app/tv-casting-common/core/CastingPlayer.cpp

```diff
@@ -210,6 +214,33 @@ void CastingPlayer::ContinueConnecting()
     }
 }
 
+void CastingPlayer::StopConnecting()
+{
+    ChipLogProgress(AppServer, "CastingPlayer::StopConnecting()");
+    CHIP_ERROR err = CHIP_NO_ERROR;
```

Comment:
Sure, these checks might be useful for debugging if the 3P developer doesn't read the ContinueConnecting() and StopConnecting() documentation. ContinueConnecting() does not require the CommissionerDeclaration callback since a reply is not expected from the CastingPlayer when ContinueConnecting() is called.